setup.cfg

```diff
@@ -1,2 +1,2 @@
 [bdist_rpm]
-requires =  f5-sdk == 1.5.0
```

Comment:
Since we are only using version 2.2 for the test, do we really need to update the version?